### PR TITLE
[WED]  v0.6.3_rdo3 to v0.6.17_rdo (RP-246) (RP-243)

### PR DESCRIPTION
**Pull request by JeanChristopheMorinRodeoFX**
| File that was conflicted |
| --- |
| hooks/scene_operation_tk-maya.py |

to make the review easier, here is what you can do:

``` bash

meld /mnt/users/jcmorin/rdoenv/tank/apps/tk-multi-workfiles_0.6.17/ /mnt/users/jcmorin/rdoenv/tank/apps/tk-multi-workfiles/
meld /mnt/users/jcmorin/rdoenv/tank/apps/tk-multi-workfiles_0.6.3/ /rodeo/repositories/tank/studio/install/apps/git/tk-multi-workfiles.git/v0.6.3_rdo5/
```

That will allow you to see a diff of **v0.6.17 <> v0.6.17_rdo1** and **v0.6.3 <> v0.6.3_rdo3**.
First _meld_ instance will allow you to see rdo cahnges in the new app, and the second _meld_ instance wil allow you to see rdo changes in v0.6.3.

So you can see if our changes are the same in the new app than in the old app, without seeing new code from Shotgun.

### CODE REVIEW

File: hooks/scene_operation_tk-maya.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ def execute(self, operation, file_path, context, parent_action, file_version, re
             # do new scene as Maya doesn't like opening 
             # the scene it currently has open!   
             cmds.file(new=True, force=True)
-            cmds.file(file_path, open=True)
+            cmds.file(file_path, open=True, force=True)
```
Comment (kookiCT) [inline hooks/scene_operation_tk-maya.py:69]: Not sure that's correct... What was the conflict exactly ?

Comment (JeanChristopheMorinRodeoFX) [inline hooks/scene_operation_tk-maya.py:69]: Here is the conflict:

``` python
        elif operation == "open":
            # do new scene as Maya doesn't like opening 
            # the scene it currently has open!   
<<<<<<< HEAD
            cmds.file(new=True, force=True)
            cmds.file(file_path, open=True)
            # add the newly opened file to the recent file list.
            pycore.mel.addRecentFile(
                file_path,
                'mayaAscii' if os.path.splitext(file_path)[-1] == '.ma' else 'mayaBinary'
            )

=======
            cmds.file(new=True, force=True) 
            cmds.file(file_path, open=True, force=True)
>>>>>>> master
        elif operation == "save":
            # save the current scene:
            cmds.file(save=True)
```

Comment (kookiCT) [inline hooks/scene_operation_tk-maya.py:69]: OK ! (although I'm not sure the force flag applies to open commands...)

**Comment by kookiCT** [comment]
:+1:

**Comment by JeanChristopheMorinRodeoFX** [comment]
I added commit from v0.6.3_rdo4 and v0.6.3_rdo5, so it is up-to-date with our changes.

**Comment by kookiCT** [comment]
:+1: from Julien

**Pull request merged**